src/topology/algebra/group.lean

```diff
@@ -330,7 +330,10 @@ class topological_add_group (G : Type u) [topological_space G] [add_group G]
   extends has_continuous_add G, has_continuous_neg G : Prop
 
 /-- A topological group is a group in which the multiplication and inversion operations are
-continuous. -/
+continuous.
+
+When you declare an instance, you should also provide an instance of `uniform_space` and
+`uniform_group` using `topological_group.to_uniform_space` and `topological_group_is_uniform`. -/
```

Comment:
Is the "you _should_ also provide" a bit too strong?  Consider, for example, `normed_uniform_group`, where the group gets a uniform structure by a different path (normed group to metric space to uniform space), so it would be wrong to provide another uniform space instance by `topological_group.to_uniform_space`.